src/talos/example_config.toml

```diff
@@ -36,25 +36,49 @@ symbol = 'AGENE'
 chrom = '1'
 
 [ValidateMOI]
-# thresholds for different filters during the MOI checks
-gnomad_dominant = 0.001
 
-# by default common in callset/gnomAD filters aren't applied to ClinVar pathogenics
-# set this flag to false to apply apply these filters
-# in testing this created a drastic reduction in candidates, at the expense of a few causative variants
-allow_common_clinvar = true
+# callset AF filtering will only be applied to variants with this AC or higher
+min_callset_ac_to_filter = 10
```

Comment:
Was this always 10 or did I bump it up, or did it used to be configured differently ant different points in the filters? Not sure what the "right" answer is, just something for us to consider.

Reply:
I think it may have been 5. I'll pull this into the main page of the report, it would be nice to see at a glance